tests/unit/storage_v3_test_utils.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include "storage_v3_test_utils.hpp"
+
+namespace memgraph::storage::v3::tests {
+
+size_t CountVertices(Storage::Accessor &storage_accessor, View view) {
+  auto vertices = storage_accessor.Vertices(view);
+  size_t count = 0U;
+  for (auto it = vertices.begin(); it != vertices.end(); ++it, ++count)
+    ;
+  return count;
```

Comment:
```suggestion
  return std::distance(vertices.begin(), vertices.end());
```

Reply:
you can also try `std::ranges::distance` but the iterator concepts have slightly changed so it might not work

Reply:
Why not call `ApproximateVertexCount`?

Reply:
@kostasrim [`std::distance`](https://en.cppreference.com/w/cpp/iterator/distance) requires iterator to be LegacyInputIterator and `std::ranges::difference` requires `std::weakly_incrementable`. Both of them are unsatisfied for this iterator. As we have to probably rewrite these iterators, I wouldn't spend time on improving them.

@jbajic I guess because it doesn't consider the transactions, meaning it might "count" vertices that are not accessible from the transaction. This iterator takes care of that.